src/main/kotlin/com/manimdsl/ManimProjectWriter.kt

```diff
@@ -16,12 +17,13 @@ class ManimProjectWriter(private val pythonCode: String) {
     }
 
     fun generateAnimation(fileName: String, options: List<String>, outputFile: String): Int {
-        val commandOptions = options.joinToString("")
-        val manimExitCode = ProcessBuilder("manim $fileName Main -$commandOptions --video_output_dir tmp".split(" "))
+        val uid = UUID.randomUUID().toString()
+        val commandOptions = options.joinToString(" ")
+        val manimExitCode = ProcessBuilder("manim $fileName Main $commandOptions --video_output_dir $uid --media_dir $uid".split(" "))
```

Comment:
should it be -$commandOptions?

Reply:
No because the list is added with "-" or "--" in Compiler.kt

Reply:
(Because there's no short form for high-quality, it has to be "--high-quality")